coordinator-client/src/contribution-data.ts

```diff
@@ -0,0 +1,5 @@
+export class ContributionData {
+    challengeHash: string
+    responseHash: string
+    contributionTime: number
```

Comment:
naming nit: "time" is used in other variable names to indicate a timestamp.  maybe call this `powersoftauDuration` or `computeDuration`?

Reply:
Will change!